Kickstarter-iOS/Views/Controllers/DiscoveryFiltersViewController.swift

```diff
@@ -32,18 +32,15 @@ internal final class DiscoveryFiltersViewController: UIViewController, UITableVi
     self.filtersTableView.dataSource = self.dataSource
     self.filtersTableView.delegate = self
 
-    self.backgroundGradientView.startPoint = CGPoint(x: 0.0, y: 1.0)
-    self.backgroundGradientView.endPoint = CGPoint(x: 1.0, y: 0.0)
-
     self.closeButton.addTarget(self, action: #selector(closeButtonTapped), for: .touchUpInside)
 
     self.viewModel.inputs.viewDidLoad()
   }
 
-  override func viewWillAppear(_ animated: Bool) {
-    super.viewWillAppear(animated)
+  override func viewDidAppear(_ animated: Bool) {
```

Comment:
yeah, i tweaked a lot of the animation because the nav header was animating some color transitions at the same time. in the end, decided we shouldn't be animating on `viewWillAppear` because it's starting the animation too early. Better to animate after `viewDidAppear` is called.